python/pyspark/streaming/dstream.py

```diff
@@ -418,10 +418,12 @@ def slice(self, begin, end):
     def _validate_window_param(self, window, slide):
         duration = self._jdstream.dstream().slideDuration().milliseconds()
         if int(window * 1000) % duration != 0:
-            raise ValueError("windowDuration must be multiple of the slide duration (%d ms)"
+            raise ValueError("windowDuration must be multiple of the parent " \
+                             "dstream's slide (batch) duration (%d ms)"
```

Comment:
dstream -> DStream
Please add [MINOR] to the title per https://spark.apache.org/contributing.html